broker/cloud_functions/lsst/ps_to_storage/main.py

```diff
@@ -1,37 +1,53 @@
 #!/usr/bin/env python3
 # -*- coding: UTF-8 -*-
 
-"""This module stores the alert data as an Avro file in Cloud Storage."""
+"""This module stores LSST alert data as an Avro file in Cloud Storage."""
 
 import base64
 import io
 import json
+import math
 import os
 import struct
 from typing import Optional
+from astropy.time import Time
 
+import flask
 import fastavro
 import pittgoogle
 from confluent_kafka.schema_registry import SchemaRegistryClient
-from google.cloud import functions_v1, logging, storage, pubsub_v1
+from google.cloud import logging, storage, pubsub_v1
 from google.cloud.exceptions import PreconditionFailed
 
+# [FIXME] Make this helpful or else delete it.
+# Connect the python logger to the google cloud logger.
+# By default, this captures INFO level and above.
+# pittgoogle uses the python logger.
+# We don't currently use the python logger directly in this script, but we could.
+logging.Client().setup_logging()
+publisher = pubsub_v1.PublisherClient()
 
 PROJECT_ID = os.getenv("GCP_PROJECT")
 TESTID = os.getenv("TESTID")
 SURVEY = os.getenv("SURVEY")
 VERSIONTAG = os.getenv("VERSIONTAG")
 
-# connect to the cloud logger and publisher
-logging_client = logging.Client()
-log_name = "ps-to-gcs-cloudfnc"
-logger = logging_client.logger(log_name)
-publisher = pubsub_v1.PublisherClient()
+# Variables for incoming data
+# A url route is used in setup.sh when the trigger subscription is created.
+# It is possible to define multiple routes in a single module and trigger them using different subscriptions.
+ROUTE_RUN = "/"  # HTTP route that will trigger run(). Must match deploy.sh
+
+# Variables for outgoing data
+HTTP_204 = 204  # HTTP code: Success
+HTTP_400 = 400  # HTTP code: Bad Request
 
 # GCP resources used in this module
 ALERTS_TOPIC = pittgoogle.Topic.from_cloud(
     "alerts", survey=SURVEY, testid=TESTID, projectid=PROJECT_ID
 )
+TOPIC_BIGQUERY_IMPORT = pittgoogle.Topic.from_cloud(
```

Comment:
```suggestion
ALERTS_TOPIC = pittgoogle.Topic.from_cloud(
    "alerts", survey=SURVEY, testid=TESTID, projectid=PROJECT_ID
)
BIGQUERY_IMPORT_TOPIC = pittgoogle.Topic.from_cloud(
```

I recommend keeping the syntax the same for clarity. (Alternately, could switch `ALERTS_TOPIC` -> `TOPIC_ALERTS`, which might actually be better both because it would match the `ROUTE_RUN` syntax and because it puts the parts in order of increasing specificity.)